src/app/services/admin-patient.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { ServerApiService } from './server-api.service';
+import { Observable } from 'rxjs';
+import { Response } from '../../../api-objects/GenericResponse';
+import { AdminPatientResponse, ResetPasswordResponse, UserListItem } from '../../../api-objects/AdminPatient';
+import { PatientResponse } from '../../../api-objects/patientResponse';
+
+@Injectable()
+export class AdminPatientService {
+  constructor(private serverApiService: ServerApiService) { }
+
+  public createPatient(patientId : string, deceased : boolean, patientNotes : string) : Observable<Response<PatientResponse>> {
+    return this.serverApiService.post('patient', {'patientId': patientId, 'deceased': deceased, 'patientNotes': patientNotes});
+  }
+
+  public getPatients() : Observable<Response<PatientResponse[]>> {
+    return this.serverApiService.get<PatientResponse[]>('patient');
+  }
+
+  public getPatient(patientId : number) : Observable<Response<PatientResponse[]>> {
+    return this.serverApiService.get<PatientResponse[]>('patient/' + patientId);
+  }
+
+  public deletePatient(patientId : number) : Observable<Response<any>> {
+    return this.serverApiService.delete<any>('patient/' + patientId);
+  }
```

Comment:
I don't like returning any.  I think this should return a boolean so we know if it's a successful delete.

Reply:
It looks like the endpoint doesn't return but may throw an error.

Reply:
Sounds like it's a `Response<null>` then.